roles/cifmw_external_dns/tasks/cert.yml

```diff
@@ -0,0 +1,83 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Put the certificate manifest file in place
+  delegate_to: localhost
```

Comment:
I guess we generate one per node - correct? If not, we can use `run_once: true`

Reply:
No, I can use `run_once`